plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/FormalVerificationPluginErrorMessages.kt

```diff
@@ -55,5 +55,11 @@ object FormalVerificationPluginErrorMessages : BaseDiagnosticRendererFactory() {
             "Function ''{0}'' may leak from its in-place contract.",
             FirDiagnosticRenderers.SYMBOL,
         )
+        put(
+            PluginErrors.CONDITIONAL_EFFECT_ERROR,
+            "Cannot verify that if {0} then ''{1}''.",
```

Comment:
IMHO the quotes around {1} don't look very neat

Reply:
Ok, I'm going to remove them!